File: app/src/main/java/ru/mamykin/foreignbooksreader/views/AboutView.java
package ru.mamykin.foreignbooksreader.views;

import com.arellomobile.mvp.MvpView;
import com.arellomobile.mvp.viewstate.strategy.AddToEndSingleStrategy;
import com.arellomobile.mvp.viewstate.strategy.StateStrategyType;

@StateStrategyType(AddToEndSingleStrategy.class)
public interface AboutView extends MvpView {
    void showAppVersion(String version);

    void openWebsite(String url);

    void showFeedbackDialog(String email, String subject, String text);

    void showChangelogDialog(String text);

    void showLibrariesDialog();

    void showToast(String text);
}

File: app/src/main/java/ru/mamykin/foboreader/presentation/about/AboutView.kt
package ru.mamykin.foboreader.presentation.about

import com.arellomobile.mvp.MvpView
import com.arellomobile.mvp.viewstate.strategy.AddToEndSingleStrategy
import com.arellomobile.mvp.viewstate.strategy.StateStrategyType

@StateStrategyType(AddToEndSingleStrategy::class)
interface AboutView : MvpView {
    fun showAppVersion(version: String)

    fun openWebsite(url: String)

    fun showFeedbackDialog(email: String, subject: String, text: String)

    fun showChangelogDialog(text: String)

    fun showLibrariesDialog()

    fun showToast(text: String)
}